backend/tests/Feature/UserPermissionsTest.php

```diff
@@ -126,4 +127,82 @@ public function testUserDoesNotHaveTestPermissionByAssignedRole()
         $user->assignRole(Role::PUBLICATION_ADMINISTRATOR);
         $this->assertFalse($user->can($this->test_permission));
     }
+
+    /**
+     * @return void
+     */
+    public function testRoleIsQueryableFromUserCreatedViaLighthouse()
+    {
+        $createUserResponse = $this->graphQL(
+            'mutation {
+                createUser(user: {
+                    email: "brandnew@gmail.com",
+                    password: "KajSu8viptUrz&",
+                    username: "testusername",
+                    name: "Test Name"
+                }) {
+                    id
+                    name
+                    username
+                }
+            }'
+        );
+        $test_role = Role::factory()->create([
+            'name' => $this->test_user_role
+        ]);
+        $user = User::where('username','testusername')->first();
+        $user->assignRole($this->test_user_role);
+        $getUserResponse = $this->graphQL(
+            'query getUser($id: ID) {
+                user(id: $id) {
+                    id
+                    name
+                    roles {
+                        id
+                        name
+                    }
+                }
+            }', ['id' => $createUserResponse["data"]["createUser"]["id"]]
+        );
+        $test_role_id = (string) $test_role->id;
+        $expected_array = [
+            0 => [
+                'id' => $test_role_id,
+                'name' => 'Test User Role'
+            ]
+        ];
+        $getUserResponse->assertJsonPath("data.user.roles", $expected_array);
+    }
+
+    /**
+     * @return void
+     */
+    public function testRoleIsQueryableFromUserCreatedViaFactory()
```

Comment:
And now I see that you have one created by the factory as well.  I guess I'm curious what we're testing differently btw these two tests.

Reply:
It's a sanity check. If a user was created via User factory or via Lighthouse, we want to ensure that the data exists and functions the same regardless.

Reply:
The duplicate test was removed.